### ISSUE TITLE
batch_norm error

### ISSUE THREAD

**Issue by qbTrible** [anonymous47823493/EagleEye#47]
Hi, when I run the search.py, an error has raised as follow, do you have any idea, thanks!
![image](https://user-images.githubusercontent.com/41276778/132642427-a4b5ec7f-5b8d-491e-893b-37981f217e43.png)

**Comment by axyz1288** [comment]
I have the same problem. It seems that the thinning does not trim the BatchNorm.
![image](https://user-images.githubusercontent.com/56338034/136000835-aa3741eb-b79b-49f9-8f67-50ecb0ef64d6.png)
